contents/tutorials/when-and-how-to-run-group-targeted-a-b-tests.md

```diff
@@ -0,0 +1,88 @@
+
+Groups can be companies, geographic locations, or any other set of users that share common characteristics.
+
+![Screenshot of setting the participant type in an experiment](../images/tutorials/when-and-how-to-run-group-targeted-a-b-tests/user-vs-group-a-b-tests.png)
+
+## When to run group-targeted experiments
+
+When the change in how an individual user uses your product will significantly affect the behavior of other users, or when you want to measure the impact of a change on an entire group of users. This is usually seen in the following types of products:
+
+1. B2B SaaS apps, in order to test how a change will affect how an entire company uses your product.
+
+   - For example, suppose Asana wants to test a new AI feature that automatically assigns tasks based on their urgency and importance. By conducting a group-targeted experiment, they can measure the impact on project completion rates across the entire company. 
+
+2. Products with network effects, since a change for one user will likely affect how other users interact with the product.
+
+   - For example, suppose Slack wants to improve the usage of a new video calling feature. Improving the feature's discoverability for a single user will increase their own usage with it, but since they use it with their coworkers, their coworkers will also discover it.
+
+## How does running a group-targeted experiment differ from a user-targeted one?
+
+There are three key differences to take into account when running a group-targeted experiment versus a user-level one:
+
+### 1. Less statiscal power
+
+Since in group-targeted experiments you treat a group of users as a single data point, you have less statistical power. For example, Slack has 20 million users, but only 600,000 companies using it.
+
+In practice, this means that you'll usually have to run group-targeted experiments longer than you would user-targeted experiments. It also means that your results may be more susceptible to noise.
+
+### 2. Higher randomization risk
+
+Randomization risk can occur when groups are not properly randomized when they are assigned to the control or test variants. Since in group-targeted experiments you have fewer data points, the risk is higher that this will occur and this can distort results.
+
+A B2B SaaS company wants to see if a new pricing plan helps retain customers. They split companies into two groups: one with the new plan and one with the old plan. However, the randomization process assigns more small companies to the new plan and more large companies to the old plan. This may skew the results because company size could be a factor in determining customer retention rather than the pricing plan itself.
```

Comment:
```suggestion
For example, a B2B SaaS company wants to see if a new pricing plan helps retain customers. They split companies into two groups: one with the new plan and one with the old plan. However, the randomization process assigns more small companies to the new plan and more large companies to the old plan. This may skew the results because company size could be a factor in determining customer retention rather than the pricing plan itself.
```